src/components/VmConsoles/VmConsoles.js

```diff
@@ -39,18 +42,49 @@ VmIsStarting.propTypes = {
   LoadingComponent: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
 };
 
+const RdpServiceNotConfigured = ({ vm }) => (
+  <div className="vm-consoles-no-rdp">
+    <ExpandCollapse
+      bordered={false}
+      align={ExpandCollapse.ALIGN_CENTER}
+      textCollapsed="No RDP Service found"
+      textExpanded="No RDP Service found"
+    >
+      <div className="vm-consoles-no-rdp-content">
+        <span>
+          This is a Windows virtual machine but no Service for the RDP (Remote Desktop Protocol) can be found.
+        </span>
+        <br />
+        <span>
+          For better experience accessing Windows console, it is recommended to use the RDP. To do so, create a service:
+          <ul>
+            <li>
+              exposing the <b>3389/tcp</b> port of the virtual machine
```

Comment:
You already have RDP constant in  https://github.com/kubevirt/web-ui/pull/129 . I propose to move it to web-ui-components and reuse it in this text.

Reply:
done